diff -ru man-pages-fr-3.45.orig/man-pages-extras-fr-0.8.1/Makefile man-pages-fr-3.45/man-pages-extras-fr-0.8.1/Makefile
--- man-pages-fr-3.45.orig/man-pages-extras-fr-0.8.1/Makefile	2008-06-03 16:13:12.000000000 +0200
+++ man-pages-fr-3.45/man-pages-extras-fr-0.8.1/Makefile	2013-01-29 12:23:40.101993080 +0100
@@ -15,138 +15,172 @@
     sgml-common switchdesk sysvinit util-linux-ng yp-tools yum man-pages-man1 \
     ntpd gzip yum-utils usbutils pciutils
 
+.PHONY : afaire
 afaire :
 	cd afaire ; \
 	make install ;
 
+.PHONY : asr-manpages
 asr-manpages :
 	cd asr-manpages-1.3 ; \
 	make install ;
 
+.PHONY : at
 at :
 	cd at-3.1.10 ; \
 	make install ;
 
+.PHONY : bash
 bash :
 	cd bash-3.2 ; \
 	make install ;
 
+.PHONY : createrepo
 createrepo :
 	cd createrepo-0.9.5 ; \
 	make install ;
 
+.PHONY : cron
 cron :
 	cd cron-4.1 ; \
 	make install ;
 
+.PHONY : diffutils
 diffutils :
 	cd diffutils-2.8.1 ; \
 	make install ;
 
+.PHONY : docbook-utils
 docbook-utils :
 	cd docbook-utils-0.6.14 ; \
 	make install ;
 
+.PHONY : eject
 eject :
 	cd eject-2.1.5 ; \
 	make install ;
 
+.PHONY : glibc-linuxthreads
 glibc-linuxthreads :
 	cd glibc-linuxthreads-2.3.6 ; \
 	make install ;
 
+.PHONY : grep
 grep :
 	cd grep-2.5.3 ; \
 	make install ;
 
+.PHONY : gzip
 gzip :
 	cd gzip-1.3.12 ; \
 	make install ;
 
+.PHONY : initscripts
 initscripts :
 	cd initscripts-8.51 ; \
 	make install ;
 
+.PHONY : lilo
 lilo :
 	cd lilo-22.8 ; \
 	make install ;
 
+.PHONY : man-pages-man1
 man-pages-man1 :
 	cd man-pages-man1-2.49 ; \
 	make install ;
 
+.PHONY : modutils
 modutils :
 	cd modutils-2.4.27 ; \
 	make install ;
 
+.PHONY : netkit-telnet
 netkit-telnet :
 	cd netkit-telnet-0.17 ; \
 	make install ;
 
+.PHONY : nfs-utils
 nfs-utils :
 	cd nfs-utils-1.1.1 ; \
 	make install ;
 
+.PHONY : ntpd
 ntpd :
 	cd ntpd-4.2.4p4 ; \
 	make install ;
 
+.PHONY : opensp
 opensp :
 	cd opensp-1.5.2 ; \
 	make install ;
 
+.PHONY : openjade
 openjade :
 	cd openjade-1.3.2 ; \
 	make install ;
 
+.PHONY : pciutils
 pciutils :
 	cd pciutils-3.0.0 ; \
 	make install ;
 
+.PHONY : procps
 procps :
 	cd procps-3.2.7 ; \
 	make install ;
 
+.PHONY : psmisc
 psmisc :
 	cd psmisc-22.6 ; \
 	make install ;
 
+.PHONY : rp-pppoe
 rp-pppoe :
 	cd rp-pppoe-3.8 ; \
 	make install ;
 
+.PHONY : sed
 sed :
 	cd sed-4.1.5 ; \
 	make install ;
 
+.PHONY : sgml-common
 sgml-common :
 	cd sgml-common-0.6.3 ; \
 	make install ;
 
+.PHONY : switchdesk
 switchdesk :
 	cd switchdesk-4.0.8 ; \
 	make install ;
 
+.PHONY : sysvinit
 sysvinit :
 	cd sysvinit-2.86 ; \
 	make install ;
 
+.PHONY : util-linux-ng
 util-linux-ng :
 	cd util-linux-ng-2.13.1.1 ; \
 	make install ;
 
+.PHONY : usbutils
 usbutils :
 	cd usbutils-0.73 ; \
 	make install ;
 
+.PHONY : yp-tools
 yp-tools :
 	cd yp-tools-2.10 ; \
 	make install ;
 
+.PHONY : yum
 yum :
 	cd yum-3.2.7 ; \
 	make install ;
 
+.PHONY : yum-utils
 yum-utils :
 	cd yum-utils-1.1.13 ; \
 	make install ;
